src/shared/components/query/QueryStore.ts

```diff
@@ -1563,7 +1563,7 @@ export class QueryStore
 		this.zScoreThreshold = params.Z_SCORE_THRESHOLD || '2.0';
 		this.rppaScoreThreshold = params.RPPA_SCORE_THRESHOLD || '2.0';
 		this.dataTypePriorityCode = params.data_priority || '0';
-		this.selectedSampleListId = params.case_set_id || "";
+		this.selectedSampleListId = params.case_set_id ? params.case_set_id.toString() : "";  // must be a string even though it's integer

```

Comment:
super crappy i know.    this should really be dealt with when we read params from query

Reply:
wait when is case_set an integer? is that for the case_set_type w_cna etc?

Reply:
oh is it for -1?

Reply:
yes